src/Card/README.md

```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 </>
 ```
 
+### Card Content Block Empty
+
+With image
+
+```jsx live
+<Card style={{ width: '25rem' }}>
+  <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" />
+  <Card.Section className="text-center">
+    <h3>Headline</h3>
+    <p>This is an optional text description.</p>
+    <Button>Action</Button>
```

Comment:
Per the Figma spec, it looks like the button is always using the brand variant. Should these examples' CTA buttons include `variant="brand"`?

Reply:
You are right. Fixed